package.json

```diff
@@ -11,16 +11,17 @@
     "lint:fix": "next lint --fix"
   },
   "dependencies": {
-    "@aleph-front/core": "^1.23.0",
+    "@aleph-front/core": "^1.23.1",
     "@aleph-sdk/account": "^1.0.3",
     "@aleph-sdk/avalanche": "^1.0.3",
+    "@aleph-sdk/base": "file:../aleph-sdk-ts/packages/base",
```

Comment:
> for testing purposes

Will set the proper version when new SDK version is released (https://github.com/aleph-im/aleph-sdk-ts/pull/182)